api/app/hfi/hfi_calc.py

```diff
@@ -110,107 +112,62 @@ def get_prep_day_dailies(dailies_date: date, area_dailies: List[StationDaily]) -
-                    'station_codes': [station_record.station_code],
-                    'station_objects': []
-                }
-            else:
-                planning_areas_dict[planning_area_record.id]['station_codes'].append(
-                    station_record.station_code)
-
-        # We're still missing some data that we need from wfwx, so give it the list of stations
-        wfwx_stations_data = await get_stations_by_codes(list(station_info_dict.keys()))
-        # Iterate through all the stations from wildfire one.
-
-        for wfwx_station in wfwx_stations_data:
-            # Combine everything.
-            station_properties = WeatherStationProperties(
-                name=wfwx_station.name,
-                elevation=wfwx_station.elevation,
-                wfwx_station_uuid=wfwx_station.wfwx_station_uuid)
-
-            for station_info in station_info_dict[wfwx_station.code]:
-                weather_station = WeatherStation(code=wfwx_station.code,
-                                                 order_of_appearance_in_planning_area_list=station_info[
-                                                     'order_of_appearance_in_planning_area_list'],
-                                                 station_props=station_properties)
-                station_info['station'] = weather_station
-                for planning_station in station_info_dict[wfwx_station.code]:
-                    existing_station_codes = [
-                        s.code for s in planning_areas_dict[planning_station['planning_area'].id]['station_objects']]
-
-                    if weather_station.code not in existing_station_codes:
-                        planning_areas_dict[planning_station['planning_area'].id]['station_objects'].append(
-                            weather_station)
-
-    # create PlanningArea objects containing all corresponding WeatherStation objects
-    for key, val in planning_areas_dict.items():
-        planning_area = PlanningArea(
-            id=val['planning_area_record'].id,
-            name=val['planning_area_record'].name,
-            order_of_appearance_in_list=val['order_of_appearance_in_list'],
-            stations=val['station_objects'])
-        val['planning_area_object'] = planning_area
-
-    # create FireCentre objects containing all corresponding PlanningArea objects
-    for key, val in fire_centres_dict.items():
-        planning_area_objects_list = []
-        for pa_record in val['planning_area_records']:
-            pa_object = planning_areas_dict.get(pa_record.id).get('planning_area_object')
-            planning_area_objects_list.append(pa_object)
-        fire_centre = FireCentre(
-            id=key, name=val['fire_centre_record'].name, planning_areas=planning_area_objects_list)
-        fire_centres_list.append(fire_centre)
-
-    return fire_centres_list
+        rows = get_fire_weather_stations(session)
```

Comment:
Can this be destructured so we don't have to reference `rows[0]`, `rows[3]` etc?

Reply:
Nevermind!

Reply:
done in: https://github.com/bcgov/wps/pull/4421/commits/76d4647b896daf69473b52ece6ab1ee71bb442fe

(some auto-save formatting in that commit as well)